tests/chain_swaps.rs

```diff
@@ -207,34 +207,25 @@ fn bitcoin_liquid_v2_chain() {
                     // This means the funding transaction was rejected by Boltz for whatever reason, and we need to get
                     // fund back via refund.
                     if update.status == "transaction.lockupFailed" {
+                        std::thread::sleep(Duration::from_secs(10));
                         log::info!("REFUNDING!");
-                        let refund_tx = BtcSwapTx::new_refund(
+                        refund_bitcoin_liquid_v2_chain(
                             lockup_script.clone(),
-                            &refund_address,
-                            &ElectrumConfig::default_bitcoin(),
-                            BOLTZ_TESTNET_URL_V2.to_owned(),
+                            refund_address.clone(),
                             swap_id.clone(),
-                        )
-                        .unwrap();
-                        let tx = refund_tx
-                            .sign_refund(
-                                &our_refund_keys,
-                                Fee::Absolute(1000),
-                                Some(Cooperative {
-                                    boltz_api: &boltz_api_v2,
-                                    swap_id: swap_id.clone(),
-                                    pub_nonce: None,
-                                    partial_sig: None,
-                                }),
-                            )
-                            .unwrap();
-
-                        refund_tx
-                            .broadcast(&tx, &ElectrumConfig::default_bitcoin())
-                            .unwrap();
-
-                        log::info!("Succesfully broadcasted claim tx!");
-                        log::debug!("Claim Tx {:?}", tx);
+                            our_refund_keys,
+                            boltz_api_v2.clone(),
+                            100,
+                        );
+                        log::info!("REFUNDING with higher fee");
+                        refund_bitcoin_liquid_v2_chain(
```

Comment:
That is nice but what I meant was a test that is not ignored but runs in CI against regtest. That new fetching logic is not trivial and we should have tests for it to make sure it works as expected in case we ever refactor that code

Reply:
Makes sense. Ended up not relying on regtest, but I think this works just as well: [0c6c3fa](https://github.com/SatoshiPortal/boltz-rust/pull/87/commits/0c6c3faf4fbac07418a4c1aa3e7d730f0a6f7e81). Let me know what you think.